agent/lib_monolog.c

```diff
@@ -306,6 +314,165 @@ static char* nr_monolog_get_context(const size_t argc,
 }
 #endif
 
+/*
+ * Purpose : Convert a zval value from context data to a nrobj_t
+ *
+ * Params  : zval
+ *
+ * Returns : nrobj_t* holding converted value
+ *           NULL otherwise
+ *
+ * Notes   : Only scalar and string types are supported.
+ *           Nested arrays are not converted and are ignored.
+ *           Other zval types are also ignored.
+ */
+nrobj_t* nr_monolog_context_data_zval_to_attribute_obj(
+    const zval* z TSRMLS_DC) {
+  nrobj_t* retobj = NULL;
+
+  if (NULL == z) {
+    return NULL;
+  }
+
+  nr_php_zval_unwrap(z);
+
+  switch (Z_TYPE_P(z)) {
+    case IS_NULL:
+      retobj = NULL;
+      break;
+
+    case IS_LONG:
+      retobj = nro_new_long((long)Z_LVAL_P(z));
+      break;
+
+    case IS_DOUBLE:
+      retobj = nro_new_double(Z_DVAL_P(z));
+      break;
+
+#if ZEND_MODULE_API_NO >= ZEND_7_0_X_API_NO /* PHP 7.0+ */
+    case IS_TRUE:
+      retobj = nro_new_boolean(true);
+      break;
+
+    case IS_FALSE:
+      retobj = nro_new_boolean(false);
+      break;
+#else
+    case IS_BOOL:
+      retobj = nro_new_boolean(Z_BVAL_P(z));
+      break;
+#endif /* PHP 7.0+ */
```

Comment:
Since we don't support any versions lower than 7.0, I think we can remove the need for this CC macro and just keep the `IS_TRUE`/`IS_FALSE` cases

Reply:
Fixed in commit [48db58c](https://github.com/newrelic/newrelic-php-agent/commit/48db58c9193277c088c67e6a07b8b698560637a8).